integrations/statuspage/pyproject.toml

```diff
@@ -6,7 +6,8 @@ authors = ["Albert Luganga <albertluganga@getport.io>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-port_ocean = {version = "^0.9.7", extras = ["cli"]}
+port_ocean = {extras = ["cli"], version = "^0.9.7"}
+
 
 [tool.poetry.group.dev.dependencies]
 pytest = "^7.2"
```

Comment:
You forgot to bump the integration version itself

[tool.poetry]
name = "statuspage"
version = "new version"